### ISSUE TITLE
Dark Theme creator/collection/tag fields are white text on white background

### ISSUE THREAD

**Issue by AevumDecessus** [manyfold3d/manyfold#4834]
**Describe the bug**
#4695 fixed the content of the dropdowns for dark themes, but once you select an item, the content of the text in the selected field is white text on a white background.

**To Reproduce**
1. Go to 'Add Content'
2. Click on 'Select a creator from the dropdown'
3. Scroll down to 'Highlight the creator field to verify one is selected'

**Expected behavior**
Text should not be white on a white background field

**Tested themes**
* Cyborg
<img width="1572" height="717" alt="Image" src="https://github.com/user-attachments/assets/80ac0a7f-95f2-49fd-8040-8f3b7ad98484" />

* Darkly
<img width="1579" height="863" alt="Image" src="https://github.com/user-attachments/assets/4bc1dba3-0541-4bd6-b68d-eff295956f8c" /> 

* Slate
<img width="1566" height="664" alt="Image" src="https://github.com/user-attachments/assets/3d608e61-a765-4da2-8f13-f2979fa0e997" />

* Solar - this one is acceptable
<img width="1574" height="712" alt="Image" src="https://github.com/user-attachments/assets/03a07394-2162-4a87-8c0d-9679348ee407" />

* Superhero
<img width="1573" height="737" alt="Image" src="https://github.com/user-attachments/assets/b2e2348f-faf8-4d8a-9713-84c6580ee86d" />

* Vapor - also acceptable
<img width="1573" height="713" alt="Image" src="https://github.com/user-attachments/assets/08b10e70-a986-4c07-bcb8-ec5e00138572" />

Screenshot has both creator and collection selected, highlighted creator field to demonstrate

** Manyfold Version
[v0.125.0](https://github.com/manyfold3d/manyfold/tree/61a0ce22127c2e3eb33948270d3197c963e14951)

**Comment by Floppy** [comment]
Thanks for the comprehensive tests! I'll do a fix.

**Comment by dlerner570** [comment]
more images from dark mode and light mode

<img width="1492" height="672" alt="Image" src="https://github.com/user-attachments/assets/d03f3ca2-d73c-4791-ada1-00f0deb12742" />
<img width="1434" height="648" alt="Image" src="https://github.com/user-attachments/assets/d6762086-b92f-42b6-b7fb-b372d02136d5" />
<img width="1462" height="631" alt="Image" src="https://github.com/user-attachments/assets/dd944dd9-5700-4192-be7d-8084ffbe2345" />

**Comment by Grecco-O** [comment]
I quickly went through the themes and following problems were found:
(Only tested on the edit model page)

Theme | Problem | Example
-- | -- | --
Cosmo | Buttons bad contrast | New Creator button in model edit
Cyborg | Dropdowns not readable | Dropdown in model edit
Darkly | Dropdowns not readable | Dropdown in model edit
Lux | Dropdowns almost not readable | Dropdown in model edit
Morph | Dropdowns/Text field almost not readable | Dropdown in model edit
Slate | Dropdowns not readable | Dropdown in model edit
Superhero | Dropdowns not readable | Dropdown in model edit
Zephyr | Buttons not readable | New Creator button in model edit

**Comment by Floppy** [comment]
Oh, fantastic, thanks @Grecco-O, that's a super useful list; I use Default, Vapor and Sketchy on my various instances, so I just didn't see these.

**Comment by Floppy** [comment]
OK, I've found a huge part of the problem; tom-select doesn't currently have anything built in for Bootstrap's automatic dark mode media query, so understandably it's all going wrong. I've done a fix for that in https://github.com/manyfold3d/manyfold/pull/4957.

Lux and Morph still have problems across ALL input fields in dark mode, not sure why, but I'll look at those now.

**Comment by Floppy** [comment]
Morph and Lux actually both have a whole heap of dark mode problems, so I'm just going to remove them.

**Comment by AevumDecessus** [comment]
Can confirm fixed in 0.128.0

<img width="1405" height="586" alt="Image" src="https://github.com/user-attachments/assets/62154140-6884-468c-a4e1-66de58328e32" />

**Comment by Floppy** [comment]
Thanks!
